docs/standard/tour.md

```diff
@@ -49,21 +49,29 @@ The following two lines both allocate memory:
 
 There is no analogous keyword to de-allocate memory, as de-allocation happens automatically when the garbage collector reclaims the memory through its scheduled run.
 
-Types within a given scope normally go out of scope once a method completes, at which point they can be collected. However, you can indicate to the GC that a particular object is out of scope sooner than method exit using the `using` statement:
+One of the less obvious but quite far-reaching features that a garbage collector enables is memory safety. The invariant of memory safety is very simple: a program is memory safe if it accesses only memory that has been allocated (and not freed). Dangling pointers are always bugs, and tracking them down is often quite difficult.
 
-[!code-csharp[MemoryManagement](../../samples/csharp/snippets/tour/MemoryManagement.csx#L6-L9)]
+The .NET runtime provides additional services, to complete the promise of memory safety, not naturally offered by a GC. It ensures that programs do not index off the end of an array or access a phantom field off the end of an object.
 
-Once the `using` block completes, the GC will know that the `stream` object in the previous example is free to be collected and its memory reclaimed.
+The following example will throw an exception as a result of memory safety.
 
-Rules for this have slightly different semantics in F#.  To learn more about resource management in F#, check out [Resource Management: The `use` Keyword](../fsharp/language-reference/resource-management-the-use-keyword.md)
+[!code-csharp[MemoryManagement](../../samples/csharp/snippets/tour/MemoryManagement.csx#L4-L5)]
 
-One of the less obvious but quite far-reaching features that a garbage collector enables is memory safety. The invariant of memory safety is very simple: a program is memory safe if it accesses only memory that has been allocated (and not freed). Dangling pointers are always bugs, and tracking them down is often quite difficult.
+## Handling unmanaged resources
 
-The .NET runtime provides additional services, to complete the promise of memory safety, not naturally offered by a GC. It ensures that programs do not index off the end of an array or accessing a phantom field off the end of an object.
+Some objects work with *unmanaged resources*: resources that are not automatically handled by the .NET runtime.  For example, @System.IO.FileStream objects operate on file handles provided by the operating system.  You will need to let the FileStream know when you are done using it, so that the FileStream can release the file handle back to the operating system.
```

Comment:
> Some objects work with *unmanaged resources*: resources that are not automatically handled by the .NET runtime.

Instead of "work with", you might say "maintain" or "reference".

I think that the two clause sentence separated by ":" isn't a good form for helping the reader. While there is a relationship between the two clauses, you need to know what they are to understand the sentence. That's not a good characteristic.

> You will need to let the FileStream know when you are done using it

While this is true, I think it is also misleading. One might think that there is a notification to the FileStream class along the lines of "I'm done using you". That's not really how the relationship works. After reading the following paragraph, I'm not sure if this one is needed, or maybe FileStream can be mentioned as an example after the more general introduction of unmanaged resources.